x/rollapp/keeper/keeper.go

```diff
@@ -104,6 +108,48 @@ func (k Keeper) mintRollappGenesisTokens(ctx sdk.Context, accounts []types.Genes
 	return nil
 }
 
+func (k Keeper) registerDenomOnBank(ctx sdk.Context, rollapp types.Rollapp, ibcDenom, accDenom string) {
```

Comment:
I'd make those arguments name more generic:
accDenom -> baseDenom
ibcDenom -> ibcBaseDenom

and than I think this function can be used in the ibc_middleware as it's doing the same logic there (so we won't have code dups)